x-pack/plugins/ml/public/application/app.tsx

```diff
@@ -81,15 +81,17 @@ const App: FC<AppProps> = ({ coreStart, deps, appMountParams }) => {
   const pageDeps = {
     history: appMountParams.history,
     setHeaderActionMenu: appMountParams.setHeaderActionMenu,
-    dataViewsContract: deps.data.dataViews,
+    dataViewsService: deps.data.dataViews,
     config: coreStart.uiSettings!,
     setBreadcrumbs: coreStart.chrome!.setBreadcrumbs,
+    contentManagement: deps.contentManagement,
```

Comment:
I reckon we should not add it to the `pageDeps`. Btw `config`, `dataViewsService` and perhaps `history` should be removed as well.

Reply:
Updated here [`e6c752f` (#159625)](https://github.com/elastic/kibana/pull/159625/commits/e6c752ff38c6e495759843261c970c3e0b5b7751)